inference-engine/thirdparty/clDNN/src/program_helpers.cpp

```diff
@@ -119,6 +119,12 @@ std::pair<bool, bool> program_helpers::are_layouts_identical(layout const& l1, l
     const auto& l2_pad = l2.data_padding;
     if (l1 == l2)
         return {true, true};
+    // If data is actually 1d along f, the layouts are identical
+    if (l1.data_type == l2.data_type && l1.size == l2.size && l1.data_padding == l2.data_padding &&
```

Comment:
I have a feeling that condition `l1.data_padding == l2.data_padding` is not enough, because padded memory, let's say, by X dimension for `bfyx` format is not exactly the same as padded memory by same dimension for blocked format (`b_fs_yx_fsv16`)

Reply:
Fixed